File: codechef/beginner/python/rearranging_digits.py
t = int(input())

while (t != 0):
    n = int(input())
    d = input()

    flag = False

    for i in range(n):
        if d[i] == "0" or d[i] == "5":
            flag = True

    t -= 1

    if flag:
        print("Yes")
    else:
        print("No")

File: codechef/beginner/cpp/rearranging_digits.cpp
#include<iostream>
using namespace std;

int main(){
    int t;
    cin>>t;

    while (t--){
        int d;
        cin>>d;
        string str;
        cin>>str;
        bool flag = false;
        for(int i=0;i<str.length();i++){
            if (str[i] == '0' || str[i] == '5'){
                flag = true;
            }
        }

        if (flag){
            cout<<"Yes"<<endl;
        }else{
            cout<<"No"<<endl;
        }
    }
    

    return 0;
}